packages/horizontal-slider-vdom/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './types';
+export * from './HorizontalSlider';
```

Comment:
We can sort those ascending like in other files.

Reply:
Fixed in: https://github.com/algolia/ui-components/pull/28/commits/e6852336f417ec12d89d5931960c83d2c7efcda0